scripts/test.sh

```diff
@@ -5,10 +5,5 @@ cp -r fixtures/2.5 fixtures/2.6
 for file in `ls fixtures/*_expected.rb` `ls fixtures/$(ruby -v | grep -o '\d\.\d')/*_expected.rb`
 do
     time ruby --disable=gems src/rubyfmt.rb `echo $file | sed s/expected/actual/` > /tmp/out.rb
-    diff /tmp/out.rb $file
-    if [[ $? -ne 0 ]]
-    then
-        echo "got diff"
-        exit 1
-    fi
```

Comment:
This conditional will exit the script early when a test fails. Is this an intentional change? Seems distinct from changing the output format.

Reply:
In my head I thought

> `diff` will exit with 1 regardless, so this is moot.

But you're right - we want to keep the short circuit. Re-added in 3384d50.